src/main/java/com/nguyenquyhy/discordbridge/DiscordBridge.java

```diff
@@ -69,11 +69,12 @@ public void onPreInitialization(GamePreInitializationEvent event) throws IOExcep
     }
 
     @Listener
-    public void onServerStart(GameStartedServerEvent event) {
-        CommandRegistry.register();
+    public void onServerStarting(GameStartingServerEvent event) {
```

Comment:
This isn't necessary, but for the sake of my next PR's please ignore this. Ill change it back after they are submitted.